dashboard/src/components/Config/ServiceBrokerList/ServiceBrokerList.test.tsx

```diff
@@ -52,6 +55,18 @@ context("if the service broker is not installed", () => {
   });
 });
 
+context("if the service brokers are accessed on an additional cluster", () => {
+  it("shows an alert with info", () => {
```

Comment:
I would find useful to also test the opposite: If it is accessed on the default cluster, the MessageAlert is not present.

I see that later on we have a test that is comparing the rendered page with  snapshot. Is it expected that test to cover the mentioned "opposite test"?

Reply:
> I would find useful to also test the opposite: If it is accessed on the default cluster, the MessageAlert is not present.

+1, added.


> 
> I see that later on we have a test that is comparing the rendered page with snapshot. Is it expected that test to cover the mentioned "opposite test"?

Well it would, but I don't like relying on snapshots for covering unit tests either - so I agree with you and just added a unit test instead.